app/controllers/games_controller.rb

```diff
@@ -0,0 +1,38 @@
+class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
+
+  def new
+    @game = Game.new
+  end
+
+  def create
+    @game = Game.create!(game_params.merge(white_player_id: current_user.id))
```

Comment:
As with `save!`, `create!` will throw an exception if there's a validation error (http://guides.rubyonrails.org/active_record_basics.html#validations), and this method is not set up to rescue from it, so you probably want to use `create` instead.